torchgeo/datasets/seco.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Sentinel 2 imagery from the Seasonal Contrast paper."""
+
+import os
+from collections import defaultdict
+from typing import Callable, Dict, List, Optional, cast
+
+import cv2
+import numpy as np
+import rasterio
+import torch
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import check_integrity, download_and_extract_archive
+
+
+class SeasonalContrastS2Dataset(VisionDataset):
+    r"""Sentinel 2 imagery from the Seasonal Contrast paper.
```

Comment:
I don't think this string contains any backslashes so it doesn't need to be a raw string